src/dataExplorer/components/ResultsPane.tsx

```diff
@@ -128,7 +149,11 @@ const ResultsPane: FC = () => {
       {
         vars: rangeToParam(range),
```

Comment:
No, not yet.

We send the vars as an extern in the payload, but it's ignored.


I was thinking of removing the vars entirely from the network request, but decided not to because:
* our 6-mo goals include an assessment of how to include variables in QxBuilder. As well as expanding SQL support. So I have no idea what that will include yet.
* the network payload size overhead is tiny. Because we only send the timeRange variables by default.